services/core-web/common/components/documents/ReplaceDocumentModal.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { FC, useState } from "react";
+
+import { Alert, Button, Col, Form, notification, Row, Typography } from "antd";
+import { MineDocument } from "@common/models/documents/document";
+import { formatDate } from "@common/utils/helpers";
+import FileUpload from "@/components/common/FileUpload";
+import { NEW_VERSION_PROJECT_SUMMARY_DOCUMENTS } from "@common/constants/API";
```

Comment:
I think that technically this file is actually usable by all documents! 🎉 
Just have a couple of files/consts named in a way that indicates they aren't.